server/cmd/server/cors_test.go

```
@@ -0,0 +1,47 @@
+	if !allowOrigin("https://app.example") {
+		t.Fatal("CORS_ORIGINS should allow the listed origin")
+	}
+}
+
+func TestCorsReflectsAllowedOrigin(t *testing.T) {
+	t.Setenv("CORS_ORIGINS", "")
+	handler := cors(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusNoContent)
+	}))
+
+	req := httptest.NewRequest(http.MethodGet, "/git/status", nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)